docs/quickstart.md

```diff
@@ -13,32 +13,463 @@ You can find the latest Sedna release [here](https://github.com/kubeedge/sedna/r
+Sedna provides two deployment methods, which can be selected according to your actual situation:
 
-### Prepare
+- Install Sedna on a cluster Step By Step: [guide here](setup/install.md).
+- Install Sedna AllinOne : [guide here](setup/local-up.md).
 
-todo
+### Component
+Sedna consists of the following components：
 
-### Configure
+![Architecture](./proposals/images/framework.png)
 
-todo
+#### GlobalManager
+* Unified edge-cloud synergy AI task management
+* Cross edge-cloud synergy management and collaboration
+* Central Configuration Management
 
-### Run
+#### LocalController
+* Local process control of edge-cloud synergy AI tasks
+* Local general management: model, dataset, and status synchronization
 
-todo
 
-### Monitor
+#### Worker
+* Do inference or training, based on existing ML framework.
+* Launch on demand, imagine they are docker containers.
+* Different workers for different features.
+* Could run on edge or cloud.
 
-todo
+
+#### Lib
+* Expose the Edge AI features to applications, i.e. training or inference programs.
+
+
+### System Design
+
+There are three stages in a [incremental learning job](./proposals/incremental-learning.md): train/eval/deploy.
+
+![](./proposals/images/incremental-learning-state-machine.png)
+
+## Deployment Guide
+
+### 1. Prepare
+
+#### 1.1 Deployment Planning
+
+In this example, there is only one host with two nodes, which had creating a Kubernetes cluster with `kind`.
+
+| NAME  | ROLES | Ip Address                    | Operating System        | Host Configuration | Storage | Deployment Module                                            |
+| ----- | ------- | ----------------------------- | ----------------------- | ------------------ | ------- | ------------------------------------------------------------ |
+| edge-node  | agent,edge   | 192.168.0.233 | Ubuntu 18.04.5 LTS  | 8C16G              | 500G    | LC，lib, inference worker |
+| sedna-control-plane | control-plane,master    | 172.18.0.2                   | Ubuntu 20.10  | 8C16G              | 500G    | GM，LC，lib，training worker，evaluate worker |
+
+#### 1.2 Network Requirements
+
+In this example the node **sedna-control-plane** has a internal-ip `172.18.0.2`, and **edge-node** can access it.
+
+### 2. Project Deployment
+
+#### 2.1 (optional) create virtual env
+
+```bash
+python3.6 -m venv venv
+source venv/bin/activate
+pip3 install -U pip
+```
+
+#### 2.2 install sedna SDK
+
+```bash
+cd $SENDA_ROOT/lib
+python3.6 setup.py bdist_wheel
+pip3 install dist/sedna*.whl
+``` 
+
+#### 2.3 Prepare your machine learning model and datasets
+
+##### 2.3.1 Encapsulate an Estimators 
+
+Sedna implements several pre-made Estimators in [example](/examples), your can find them from the python scripts called `interface`.
+Sedna supports the Estimators build from popular AI frameworks, such as TensorFlow, Pytorch, PaddlePaddle, MindSpore. Also Custom estimators can be used according to our interface document. 
```

Comment:
Custom estimator is important, but it can take quite a few effort in the "Quick Start" and not needed to detail it with so much space here: a link is already sufficient.

Reply:
> Custom estimator is important, but it can take quite a few effort in the "Quick Start" and not needed to detail it with so much space here: a link is already sufficient.

`estimator` is an abstract concept that's been introduced by `sedna`, without detail, the fear is that it will not be understood.